dashboard/modules/log/log_head.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+import mimetypes
+
+import aiohttp.web
+import ray.new_dashboard.utils as dashboard_utils
+from ray.new_dashboard.datacenter import DataSource, GlobalSignals
+
+logger = logging.getLogger(__name__)
+routes = dashboard_utils.ClassMethodRouteTable
+
+
+class LogHead(dashboard_utils.DashboardHeadModule):
+    LOG_URL_TEMPLATE = "http://{ip}:{port}/logs"
+
+    def __init__(self, dashboard_head):
+        super().__init__(dashboard_head)
+        mimetypes.add_type("text/plain", ".err")
+        mimetypes.add_type("text/plain", ".out")
+        routes.static("/logs", self._dashboard_head.log_dir, show_index=True)
+        GlobalSignals.node_info_fetched.append(
+            self.insert_log_url_to_node_info)
+
+    async def insert_log_url_to_node_info(self, node_info):
+        ip = node_info.get("ip")
+        if ip is None:
+            return
+        agent_port = DataSource.agents.get(ip)
+        if agent_port is None:
+            return
+        agent_http_port, _ = agent_port
+        log_url = self.LOG_URL_TEMPLATE.format(ip=ip, port=agent_http_port)
```

Comment:
Can you just use fstring? We are trying to move onto fstring in general. I think it is fine not to use template because it is used only for 3 places.

Reply:
So, `ray` has dropped support for Python 3 - 3.6? I prefer to use fstring than old formatter.

Reply:
It dropped the support for 3-3.5. Afaik, 3.6 supports fstring?

Reply:
Yes, Python 3.6 supports fstring. I am happy to know that I can use fstring.